gamla/url_utils.py

```diff
@@ -0,0 +1,21 @@
+from typing import Dict, Text
+from urllib import parse
+
+from gamla import currying, functional_generic
+
+
+@currying.curry
+def add_to_query_string(params_to_add: Dict, url: Text):
+    """
+    Add params_to_add to the query string part of url
+
+    >>> add_to_query_string({ "param1" : "value"}, "http://domain.com")
+    http://domain.com?param1=value
+    """
+    (scheme, netloc, path, query, fragment) = parse.urlsplit(url)
+
+    new_query = functional_generic.merge(parse.parse_qs(query), params_to_add)
```

Comment:
maybe pipe?

Reply:
it's a problem because merge accepts *args so I would need a lambda. not sure it will look better in this case